library/mariadb

```diff
@@ -8,11 +8,11 @@ Tags: 10.3.0, 10.3
 GitCommit: f76084f0f9dc13f29cce48c727440eb79b4e92fa
 Directory: 10.3
 
-Tags: 10.2.6, 10.2
+Tags: 10.2.6, 10.2, 10, latest
 GitCommit: 60e0d2d15e74f6fed5b6cc0c2cab4b3f9def6e6e
 Directory: 10.2
 
-Tags: 10.1.24, 10.1, 10, latest
+Tags: 10.1.24, 10.1
```

Comment:
I've been intentionally skipping this change while the stretch bits got worked out (on my phone on PTO so don't have the issue link handy), but it's been long enough that we should likely give up on stretch for 10.2 for now and just move ahead. 👍